### DISCUSSION TITLE
Onthis - Shortcuts

### DISCUSSION THREAD

**Discussion by igoryuzo** [ethereum-optimism/developers#63] [Show and tell]
### Description

Onthis builds Shortcuts, which allow users to send ETH to human-readable addresses to bridge, swap, stake, mint, and more
directly from your wallet.

For Optimism we built a few Shortcuts, one I'll share now is an easy bridge to Optimism.

Simply send ETH from mainnet -> "toOptimism.eth" directly from Metamask and see your funds arrive on Optimism.

This UX is simpler and safer.
- no phishing from malicious UIs
- less actions which save time
- easy to do on mobile
- no geofencing

Important Links
[Site](https://onthis.xyz/)
[Twitter](https://twitter.com/onthisxyz)
[Docs](https://docs.onthis.xyz/overview/)
[Analytics](https://dune.com/Nodar/onthisxyz)

**Comment by sbvegan** [comment]
Cool! Thanks for sharing :)
